tests/test_crossattention.py

```diff
@@ -31,25 +32,28 @@
         for num_heads in [4, 6, 8, 12]:
             for rel_pos_embedding in [None, RelPosEmbedding.DECOMPOSED]:
                 for input_size in [(16, 32), (8, 8, 8)]:
-                    test_case = [
-                        {
-                            "hidden_size": hidden_size,
-                            "num_heads": num_heads,
-                            "dropout_rate": dropout_rate,
-                            "rel_pos_embedding": rel_pos_embedding,
-                            "input_size": input_size,
-                        },
-                        (2, 512, hidden_size),
-                        (2, 512, hidden_size),
-                    ]
-                    TEST_CASE_CABLOCK.append(test_case)
+                    for flash_attn in [True, False]:
+                        test_case = [
+                            {
+                                "hidden_size": hidden_size,
+                                "num_heads": num_heads,
+                                "dropout_rate": dropout_rate,
+                                "rel_pos_embedding": rel_pos_embedding if not flash_attn else None,
+                                "input_size": input_size,
+                                "use_flash_attention": flash_attn,
```

Comment:
Hi @virginiafdez , do you think it would make sense to compare the output consistency with versus without `use_flash_attention`?

Reply:
@ericspod should we add more use cases here?

Reply:
I'm working on the consistency but we've seen that results are different with and without flash attention, and with CPU vs GPU.

Reply:
Will try add more tests in https://github.com/Project-MONAI/MONAI/pull/7996